packages/@guardian/source-react-components-development-kitchen/src/components/toggle-switch/styles.ts

```diff
@@ -3,9 +3,12 @@ import { neutral, success } from '@guardian/src-foundations';
 import { textSans } from '@guardian/src-foundations/typography';
 
 export const toggleSwitchStyles = css`
+	display: flex;
+	align-items: center;
 	button {
```

Comment:
This is outside the scope of this PR but in general I'd avoid using CSS selectors like this. I find it's easier to scale css when you apply just the css for each element to itself, and don't use selectors to target beyond that. Eg. Have a `buttonStyles` piece of css.

The exception to this is when it's hard to target something because it's buried inside an external component (which might be the case here!)

Reply:
One reason I did this was that I wanted to have different style for `slim` & `medium` but I thought maybe it's better not to do this kind of check multiple times `isSlim ? slimStyles : mediumStyles,` once for the container div and once for the button. 

Also I wanted to add `aria-checked` style to button but wasn't sure how to do it in emotion. 
```
button[aria-checked='false'] {
    background-color: rgba(153, 153, 153, 0.5);
}
```
or 
```
button[aria-checked='true']:after {
  left: 20px;
  background: ${neutral[100]};
}
```

Reply:
The Emotion way of selecting based on attributes:

```tsx
const buttonStyles = css`
 /* applied directly to the button */
  width: 3.188rem;
  height: 1.938rem;
  border-radius: 15.5px;

 /* applied to the button when not checked */
  &[aria-checked='false'] {
    background-color: rgba(153, 153, 153, 0.5);
  }

 /* applied to the :after pseudo-element when the button is checked */
  &[aria-checked='true']:after {
    left: 20px;
    background: ${neutral[100]};
  }
`
```

Reply:
Thank you both for giving me the idea. it's all great learning for me here :) I moved all the button styles to the button element